Makefile

```diff
@@ -403,11 +403,12 @@ _build/NoInit.proxy: _build/Base.proxy
 	touch $@
 
 _build/PoplogCommander.proxy: _build/Stage2.proxy
-	mkdir -p _build/cmdr
-	GETPOPLOG_VERSION="$(GETPOPLOG_VERSION)" sh makePoplogCommander.sh > _build/cmdr/poplog.c
-	( cd _build/cmdr && $(CC) $(CFLAGS) -Wextra -Werror -Wpedantic -o poplog poplog.c )
+	mkdir -p _build/commander
+	mkdir -p _build/poplog_base/pop/bin
+	( GETPOPLOG_VERSION="$(GETPOPLOG_VERSION)" bash makePoplogCommander.sh ) > _build/commander/poplog.c
```

Comment:
Why is the first command in a subshell? Each line in a makefile is run in its own subshell (https://www.gnu.org/software/make/manual/html_node/Execution.html#Execution); this seems unnecessary.

Reply:
I think this was left over from a simplification. I will fix.

Reply:
I fixed this and an adjacent line.